sdk/spring/scripts/Merge-Main.ps1

```diff
@@ -0,0 +1,42 @@
+[CmdLetBinding()]
+param(
+    [string]$Source = 'main',
+    [string[]]$Theirs = @('**'), # paths to always overwrite
+    [string[]]$Ours = @('sdk/spring'), # paths to never merge or overwrite
```

Comment:
`spring-experimental` should be excluded like `spring`, the new service directory `sdk/jdbc` is coming in the main branch, which may be excluded later.

Reply:
I added spring-experimental to $Ours.